lib/ui/page/home/page/my_profile/controller.dart

```diff
@@ -182,8 +176,34 @@ class MyProfileController extends GetxController {
   /// Returns the current [Credentials].
   Rx<Credentials?> get credentials => _authService.credentials;
 
+  /// [ScrollController] passed to [ListView.builder] that is attached to
+  /// [ListObserverController] to pass to a [Scrollbar].
+  ScrollController get scrollController => observerController.controller!;
+
+  /// Callback getter passed to [ListViewObserver].
+  Function(ListViewObserveModel resultModel) get onObserve => (resultModel) {
+        if (_isIgnoreObserving) {
+          Future.delayed(300.milliseconds, () => _isIgnoreObserving = false);
+          return;
+        }
+
+        final showedChild = resultModel.displayingChildModelList.first;
+
+        final ProfileTab tab = ProfileTab.values[showedChild.index];
+        if (router.profileSection.value != tab) {
+          _lastSelectedProfileTab = tab;
+          router.profileSection.value = tab;
+        }
+      };
+
+  /// Last selected [ProfileTab].
+  ProfileTab? _lastSelectedProfileTab;
+
   @override
   void onInit() {
+    router.profileSection.value =
+        ProfileTab.values[observerController.initialIndex];
+
```

Comment:
@SleepySquash , It's additional security check i think, cause If controller would be reinited somehow at any case at lifecycle of the app, then initialIndex would be set as the index of currently active tab and we will save the state of the currently selected tab as it is and won't reset it to the first tab at reIniting, creating sync problem with profile view and tab view. As for default, initial index equals 0 if no profileSection value exists already, so at first init it'll select the first tab by default.

Reply:
@SleepySquash , after rethinking it one more time i understood that it would be better just to do `router.profileSection.value ??= ProfileTab.values.first;` , behavior won't change, but we won't reassign this value on every reInit of the controller, for example when entering and leaving page, but assign value only when selected tab is **NULL**, that could only be at the first time we were entering this page and no tab were selected before.

fixed on 30a07990e4ee9b6e1b585cf38fe90281721bef98